yarn.lock

```diff
@@ -0,0 +1,216 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This project does not yet support yarn, so this should be removed from this PR.

Reply:
Oops, didn't mean to commit that, thanks